### ISSUE TITLE
Can we disable the one-time questionnaire for the crowstudy server?

### ISSUE THREAD

**Issue by crescendochu** [ProjectSidewalk/SidewalkWebpage#3357]
The questions are covered in the pre/post questionnaire for the study.
<img width="635" alt="image" src="https://github.com/ProjectSidewalk/SidewalkWebpage/assets/60413478/fbffa49e-7502-49e1-8298-59f302577943">

**Comment by jonfroehlich** [comment]
Good point. Don't need this for the study. @misaugstad, easy to take out for prediction model study?
